projects/data-grid/src/lib/components/grid-header/grid-header.component.html

```diff
@@ -1,13 +1,11 @@
-<ngrx-grid-header-item *ngIf="checkboxSelection">
-  <div class="data-grid-checkbox-wrapper data-grid-checkbox">
-    <input
-      type="checkbox"
-      [checked]="allSelected"
-      (change)="onToggleSelectAllRows()"
-      id="select-all"
-      class="data-grid-input"/>
-    <label class="data-grid-label" for="select-all"></label>
-  </div>
+<ngrx-grid-header-item *ngIf="selectionType">
+  <ngrx-row-select
```

Comment:
View is not broken when selection type is radio button, right?

Reply:
Yes. It looks the same as the checkbox selection.